execute_test.go

```diff
@@ -23,6 +23,17 @@ func (f roundTripFunc) RoundTrip(req *http.Request) (*http.Response, error) {
 	return f(req), nil
 }
 
+type testQuery struct {
+	queryer               graphql.Queryer
+	t                     *testing.T
+	expectedOperationName string
+}
+
+func (q *testQuery) Query(ctx context.Context, input *graphql.QueryInput, receiver interface{}) error {
+	assert.Equal(q.t, input.OperationName, q.expectedOperationName)
```

Comment:
Is it better to do this than just do assert.Equal calls inside the test (e.g. like the assertion for hello world bellow on line 1586?

Reply:
Ahh that's what I was missing. Yes that makes more sense